Objects/TI4_BuildArea.ttslua

```diff
@@ -653,7 +653,7 @@ function getProduceConsumeItems()
             for _, object in ipairs(getAllObjects()) do
                 maybeAdd(object)
             end
-            local guidToColor = _zoneHelper.zonesFromPositions()
+            local guidToColor = _zoneHelper.zonesFromPositions(guidToPosition)
```

Comment:
Hmm, zonehelper zonesFromPositions asserts the arg is a table.  How was this not failing before?

Reply:
This was just buried behind an IF statement, so it didn't come up until someone tried putting AIDA into the build area.